app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Users#index</h1>
```

Comment:
Lets rename heading like `Users Listing`

Reply:
i still have to implement this task i-e the user listings task. I'll update it.